src/vivarium_gates_nutrition_optimization/components/observers.py

```diff
@@ -109,8 +137,26 @@ def register_observations(self, builder: Builder) -> None:
             aggregator=partial(aggregate_state_person_time, builder.time.step_size()()),
         )
 
+    def format(self, measure: str, results: pd.DataFrame) -> pd.DataFrame:
+        results = results.reset_index()
+        results.rename(columns={"maternal_bmi_anemia_category": "sub_entity"}, inplace=True)
+        return results
+
+    def get_measure_column(self, measure: str, results: pd.DataFrame) -> pd.Series:
```

Comment:
Just a comment and Ill need to track down the VPH/vivarium PR where this is happening but I feel like you could wrap all these methods in a way that would make your life easier but Im also assuming you tried to do that.

Reply:
> Just a comment and Ill need to track down the VPH/vivarium PR where this is happening but I feel like you could wrap all these methods in a way that would make your life easier but Im also assuming you tried to do that.

@albrja Not totally sure what you mean but I agree this isn't great and I'd love ideas. But there _are_ defined in vph with sensible defaults. It's just that the defaults aren't so sensible that they usually seem to require overwriting